src/app/api/scheduling.api.ts

```diff
@@ -0,0 +1,102 @@
+import { MutationResult } from "apollo-angular";
+import { Injectable } from "@angular/core";
+import {
+    GetDatasetBatchingGQL,
+    GetDatasetBatchingQuery,
+    GetDatasetIngestScheduleGQL,
+    GetDatasetIngestScheduleQuery,
+    ScheduleInput,
+    SetConfigBatchingGQL,
+    SetConfigBatchingMutation,
+    SetConfigScheduleGQL,
+    SetConfigScheduleMutation,
+    TimeDeltaInput,
+} from "./kamu.graphql.interface";
+import { Observable, first, map } from "rxjs";
+import { ApolloQueryResult } from "@apollo/client";
+import { DatasetOperationError } from "../common/errors";
+
+@Injectable({ providedIn: "root" })
+export class SchedulingApi {
+    constructor(
+        private getDatasetIngestScheduleGQL: GetDatasetIngestScheduleGQL,
+        private getDatasetBatchingGQL: GetDatasetBatchingGQL,
+        private setConfigScheduleGQL: SetConfigScheduleGQL,
+        private setConfigBatchingGQL: SetConfigBatchingGQL,
```

Comment:
Word 'dataset' was good in `get`, but '`set` part lost it.

Reply:
Renamed